docs/topics_5/proc_web-obtaining-latest-image.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide_5/master.adoc
+
+
+[id="proc_web-obtaining-latest-image_{context}"]
+= Obtaining the latest image
+
+The first step in troubleshooting in an OpenShift Container Platform environment is to be sure you are using the latest image.
+
+NOTE: If you have deployed the {WebName} by pasting in a JSON template, no image stream is created and the following steps are not applicable.
```

Comment:
@jonathanvila  Does this note mean that the "latest image" section only applies to Operator installations?

Reply:
@jonathanvila the question in about section 2.4.1

https://access.redhat.com/documentation/en-us/migration_toolkit_for_applications/5.1/html/web_console_guide/installing_the_web_console#troubleshoot_web_console_openshift_install_ocp-311

Reply:
Regarding the operator, the user should not manage any MTA version, as the operator is the one who has to take care of that (upgrading, etc)

Reply:
@jonathanvila -- so it sounds like we should take this section out. The note says it does not apply in the case of a template and you say the user does not manage the MTA version when they install with the operator -- so is there a case where a user should do this procedure?